docs/components/movement-sensor/accel-adxl345-micro-rdk.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Configure an ADXL345 Accelerometer (Micro-RDK)"
+title: "ADXL345 Accelerometer (viam-micro-server)"
```

Comment:
Why is the "Configure a" going? We have those on all model pages. Is the title too long? If so remove accelerometer I think